libvast/vast/access.hpp

```diff
@@ -8,76 +8,34 @@
 
 #pragma once
 
-#include <type_traits>
-
 namespace vast {
 
 /// Wrapper to encapsulate the implementation of concepts requiring access to
 /// private state.
 struct access {
```

Comment:
Something for the future: this type exists only to give some parsers/printers access to the private state of befriended types. It's redundant to our `inspect` overloads, which we should use instead. `git grep` reveals only `address`, `subnet`, and `pattern` still using this.

Reply:
Thanks for the insight. We should clean that up in a follow-up PR then.